### ISSUE TITLE
reported syntax errors with multi-line pipe-style (`|`) strings

### ISSUE THREAD

**Issue by FauxFaux** [vermut/intellij-ansible#70]
A document with pipe/bar/`|`/... style multi-line strings is reported as covered in errors, it seems. It looks like this is supposed to work, because I see it being fixed before in the changelog.

Example syntax:
```yaml
foo: |
  bar
```

A screenshot of it being highlighted as an error:
![broken indentation](https://i.imgur.com/bDkolu5.png)

**Comment by vermut** [comment]
True, it mistakes PGP block for Yaml header. Will see if it can be fixed.

**Comment by funivan** [comment]
Also, yaml supports `|-`
![08 32 14](https://user-images.githubusercontent.com/425208/39979831-5eaede5e-5751-11e8-82a0-95b590bbddfe.png)

**Comment by wbkoetsier** [comment]
I'm using this plugin on PyCharm to get YAML syntax highlighting, including spotting errors.

I realise that this is an old issue and that this plugin is not under active development. I'd like to report this anyway, who knows someone likes to fix.

I'm also experiencing a problem with multi line strings, either using `|` or `>`.

```
examples:
  - no errors here
  - 'no erreors {here} too+'
  - >
    this is not allowed by the plugin, but is
    correct YAML syntax
    even with + or { or whatever.
```

YAML: 1.2
PyCharm: 2016.3 (still on 2016 due to a debugger related bug in 2017 version...)
YAML/Ansible plugin: 0.9.5

**Comment by gerhard4** [comment]
Same for `>` and `>-` strings. See also https://yaml-multiline.info/ for the kinds of strings YAML supports.
